src/zope/component/standalonetests.py

```diff
@@ -1,13 +1,14 @@
 """
 See: https://bugs.launchpad.net/zope3/+bug/98401
 """
-import sys
-import pickle
 
-def write(x): # pragma: NO COVER
+import sys # pragma: no cover (runs in subprocess)
+import pickle # pragma: no cover (runs in subprocess)
+
+def write(x): # pragma: no cover (runs in subprocess)
```

Comment:
ISTM it would be better just to [exclude the whole module via `.configrc`](http://coverage.readthedocs.io/en/coverage-4.2/source.html#execution), e.g.:

```ini
[run]
omit =
    src/zope/component/standalonetests.py
```